common/templates/darklang/preview_lang_include.html

```diff
@@ -0,0 +1,35 @@
+<%!
+from django.utils.translation import ugettext as _
+%>
+<%page expression_filter="h"/>
+
+<h1>
+    ${_("Preview Language Setting")}
+</h1>
+<div >
+    <form class="form" action="/update_lang/" method="post">
+        <fieldset class="form-group">
+            <div class="field">
+                <label class="field-label">${_("Language Code")}
+                    <input class="field-input input-text" type="text" name="preview_lang" placeholder="e.g. en" />
```

Comment:
The placeholder text should also be internationalized-- all user-visible text (including screenreader text) should be internationalized.